en/components/icon.md

```diff
@@ -75,6 +75,41 @@ You can set the icon's size through CSS. Create a custom CSS class and name it *
     <iframe id="icon-sample2-iframe" seamless width="100%" height="100%" frameborder="0" src="{environment:demosBaseUrl}/icon-sample2"  onload="onSampleIframeContentLoaded(this);">
 </div>
 
+You can also use SVG image as an icon. First, you need to inject [`IgxIconService`]({environment:angularApiUrl}/classes/igxiconservice.html) dependency in a component's constructor. Use the [`addSvgIcon`]({environment:angularApiUrl}/classes/igxiconservice.html#addsvgicon) method to import the SVG file in cache. Icon name and file URL path are method's mandatory parameters, you can specify font-set as well. After that, you can use the SVG files in the HTML markup. Alternatively, you can use the `addSvgIconFromText` method to import the SVG file providing the SVG text content instead of the file URL.
```

Comment:
You might note here that an SVG could be cached anywhere in the application and could be used from any component after being cached.